view/frontend/web/instantsearch.js

```diff
@@ -207,408 +206,454 @@ requirejs(['algoliaBundle','Magento_Catalog/js/price-utils'], function(algoliaBu
+
+						return hoganTemplate.render(_data)
+					}
 				}
 			},
 			/**
-			 * Sorting
-			 * Docs: https://community.algolia.com/instantsearch.js/v2/widgets/sortBySelector.html
+			 * sortBy
+			 * Docs: https://www.algolia.com/doc/api-reference/widgets/sort-by/js/
 			 **/
-			sortBySelector: {
+			sortBy: {
 				container: '#algolia-sorts',
-				indices: algoliaConfig.sortingIndices,
-				cssClass: 'form-control'
+				items: algoliaConfig.sortingIndices.map(function (sortingIndice) {
+					return {
+						label: sortingIndice.label,
+						value: sortingIndice.name,
+					}
+				})
 			},
 			/**
-			 * Widget name: Current refinements
+			 * currentRefinements
 			 * Widget displays all filters and refinements applied on query. It also let your customer to clear them one by one
-			 * Docs: https://community.algolia.com/instantsearch.js/v2/widgets/currentRefinedValues.html
+			 * Docs: https://www.algolia.com/doc/api-reference/widgets/current-refinements/js/
 			 **/
-			currentRefinedValues: {
+			currentRefinements: {
 				container: '#current-refinements',
-				cssClasses: {
-					root: 'facet'
-				},
 				templates: {
-					header: '<div class="name">' + algoliaConfig.translations.selectedFilters + '</div>',
-					clearAll: algoliaConfig.translations.clearAll,
 					item: $('#current-refinements-template').html()
 				},
-				attributes: attributes,
-				onlyListedAttributes: true
+				panelOptions: {
+					templates: {
+						header: '<div class="name">' + algoliaConfig.translations.selectedFilters + '</div>'
+					}
+				},
+				includedAttributes: attributes.map(function (attribute) {
+					return attribute.name
+				}),
+				transformItems: function (items) {
+					return items.map(function (item) {
+						var attribute = attributes.filter(function (_attribute) {
+							return item.attribute === _attribute.name
+						})[0];
+						if (!attribute) return item;
+						return {...item, label: attribute.label}
+					})
+				}
+			},
+
+			/**
+			 * clearRefinements
+			 * Widget displays a button that lets the user clean every refinement applied to the search. You can control which attributes are impacted by the button with the options.
+			 * Docs: https://www.algolia.com/doc/api-reference/widgets/clear-refinements/js/
+			 **/
+			clearRefinements: {
+				container: '#clear-refinements',
+				templates: {
+					resetLabel: algoliaConfig.translations.clearAll,
+				},
+				includedAttributes: attributes.map(function (attribute) {
+					return attribute.name
+				}),
+				transformItems: function (items) {
+					return items.map(function (item) {
+						var attribute = attributes.filter(function (_attribute) {
+							return item.attribute === _attribute.name
+						})[0];
+						if (!attribute) return item;
+						return {...item, label: attribute.label}
+					})
+				}
+			},
+			queryRuleCustomData: {
+				container: '#algolia-banner',
+				templates: {
+					default: `
```

Comment:
can you use template strings here? I haven't checked other places, but IIRC there's no babel on this project